cypress/integration/my_page.spec.ts

```diff
@@ -5,6 +5,8 @@ describe('/my', () => {
     let dummyFollowing : {data: ZetkinMembership[]};
     let dummyCampaigns : {data: ZetkinCampaign[]};
 
+    const date = new Date();
```

Comment:
I don't think this should be a "global" variable like this. The `Date` class is mutable, meaning that someone could accidentally (or without realizing it's consequences) change the value of this `date` to something other than today, causing subsequent tests to fail.

Instead, just instantiate the date when you need it.